addons/custom-zoom/addon.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "Custom Zoom Settings",
+  "description": "Adjust the minimum, maximum, speed, and start size of the zoom in the project editor, and autohide the controls.",
```

Comment:
Both the title and the description should probably clarify that this is about block zoom, because zoom also exists in the paint editor.

Reply:
Would "workspace" be better? or stick with block?

Reply:
How about 
```json
{
  "name": "Custom Workspace Zoom Settings",
  "description": "Adjust the minimum, maximum, speed, and start size of the zoom in the project code editor, and autohide the controls."
}
```